cypress/integration/cucumber/steps_definition/login.ts

```diff
@@ -0,0 +1,22 @@
+import {Given, When, Then} from "cypress-cucumber-preprocessor/steps";
+/// <reference types="../../../support" />
+
+Given("User tries to visit collections page", async () => {
+  cy.visit("/user/collections/");
+});
+
+Given("User is redirected to login page", async () => {
```

Comment:
Refactor `Given` to `And` to ensure your step definition is mapped to the right step written in the feature file